lib/components/MusicScreen/album_item.dart

```diff
@@ -214,14 +228,33 @@ class _AlbumItemState extends State<AlbumItem> {
                   title: Text(local.shuffleToQueue),
                 ),
               ),
+              isDownloadRequired
```

Comment:
This flag is true if the download is required by the user, as opposed to not downloaded or incidentally downloaded.  Do you have a suggestion for a better name?  Maybe isDownloadLocked or something?

Reply:
isAlwaysDownloaded, isDownloadPersisted, manuallyDownloaded, explicitlyDownloaded.

Something like that, but the current name is ambiguous and I interpreted it as "item still needs to be downloaded because it hasn't been downloaded yet" :)

Reply:
Changed to explicitlyDownloaded.